src/sections/Article/index.tsx

```diff
@@ -61,7 +63,10 @@ const Article = ({
 }: Props): JSX.Element => (
   <article className={styles.article}>
     {childrenPosition === 'before' && children && <div>{children}</div>}
-    <h1 className={styles.headline}>{title}</h1>
+    <header>
+      <h1 className={styles.headline}>{title}</h1>
+      <p>{description}</p>
```

Comment:
since we are optionally using `description`, this will not render empty `p` tag if no description is available
```suggestion
    {description && <p>{description}</p>}
```